File: c_Programming/Assingements/Assignment 9/findEven_odd.c
#include<stdio.h>
void input(int*arr,int size);
void display(int*arr,int size);
void findEven_Odd(int*arr,int size);
void main()
{
	int arr[10];
	input(arr,10);       
	display(arr,10);
	findEven_Odd(arr,10);
}
	
void input(int*arr,int size)
{
    	printf("Enter 10 elements:\n");
    for(int i=0; i<10; i++)
    {
        scanf("%d",&arr[i]);       // print the value given from the user 
    }
}
void display(int*arr,int size)
{
	printf("\nArray = [ ");
    for(int i=0; i<10; i++)
    {
        printf("%d ,",arr[i]);  ///printing the array like array=[1,2,3....]
    }
    printf("\b]\n");
}
void findEven_Odd(int*arr,int size)
{
	printf("\neven number :  [ ");
    for(int i=0; i<10; i++)
    {
    	if(arr[i]%2==0)               //even number  code
        printf("%d ,",arr[i]); 
    }
    printf("\b]\n");
    printf("\nodd number :  [ ");             
    for(int i=0; i<10; i++)
    {
    	if(arr[i]%2 !=0)               //odd  number code 
        printf("%d ,",arr[i]); 
    }
    printf("\b]\n");
}   
    

File: Java_Module/Assignment/Assignment6/ArrayProgram/src/FindEvenOdd.java
import java.util.Scanner;
class FindEvenOdd {

	public static void main(String[] args) {
		Scanner sc =new Scanner(System.in);
		int[]arr= new int[10];
		
		//Input Array
		System.out.println("Enter a 10 element : ");
		for(int i=0; i<arr.length; i++) {
			arr[i]=sc.nextInt(10);
		}
		
		//Display Array
		System.out.print("Array = [");
		for(int i=0; i<arr.length; i++) {
			System.out.print(arr[i] + " ");
		}
		System.out.print("]");
		System.out.println();
		
		//Find Even And Odd number in the Array
		System.out.print("Even Number : [ ");
		for(int i=0; i<arr.length; i++) {
			if(arr[i]%2==0)
				System.out.print(arr[i] + " ");
		}
		System.out.print("]");
		System.out.println();
		System.out.print("Odd Number : [ ");
		for(int i=0; i<arr.length; i++) {
			if (arr[i]%2 != 0)
				System.out.print(arr[i] + " ");
		}
		System.out.print("]");
		System.out.println();
		sc.close();

	}

}
